src/Utils/Authenticator.php

```diff
@@ -59,7 +59,14 @@ public function authenticate( stdClass $user ): WP_User {
 		}
 
 		if ( email_exists( $user->email ) ) {
-			return get_user_by( 'email', $user->email );
+			$user_wp = get_user_by( 'email', $user->email );
+
+			/**
+			 * Fires once the user has been authenticated.
+			 */
+			do_action( 'rtcamp.google_user_login', $user_wp, $user );
```

Comment:
From action name, it's not identified that this action will execute once user is authenticated. It only from action documentation. Maybe something like below examples.

`rtcamp.google_user_authenticated` or `rtcamp.google_user_logged_in`